hail/python/hail/docs/guides/genetics.rst

```diff
@@ -315,6 +315,34 @@ Multiple Phenotypes
             patterns of missingness. Approach #2 will do two passes over the data while Approaches #1 and #3 will
             do one pass over the data and compute the regression statistics for each phenotype simultaneously.
 
+Using Variants (SNPs) as Covariates
++++++++++++++++++++++++++++++++++++
+
+:**tags**: sample genotypes covariate
+
+:**description**: Use sample genotype dosage at specific variant(s) as covariates in regression routines.
+
+:**code**:
+
+    Convert a Python list of variant strings to a Hail set literal:
+
+    >>> my_snps = ['1:1:A:T', '16:29102912:T:TTT']
+    >>> mt_snps_lit = hl.literal({hl.parse_variant(x) for x in my_snps})
+
+    Compute an array of genotypes per sample and annotate the matrix table `mt`:
+
+    >>> mt_filt = mt.filter_rows(mt_snps_lit.contains(mt.row_key))
+    >>> sample_genos = mt_filt.annotate_cols(genotypes = hl.agg.collect(mt_filt.GT.n_alt_alleles()))
```

Comment:
What is the type of `genotypes`? How are both variants accounted for?

Reply:
this is an array